azext_edge/edge/providers/check/dataflow.py

```diff
@@ -342,16 +483,87 @@ def _process_dataflow_destinationsettings(
 
 
 def _process_endpoint_authentication(
-    endpoint_settings: dict, check_manager: CheckManager, target: str, namespace: str, padding: int
+    endpoint_settings: dict, check_manager: CheckManager, target: str, namespace: str, padding: int, detail_level: int
 ) -> None:
-    auth = endpoint_settings.get("authentication", {})
+    auth_property_dict = {
+        DataFlowEndpointAuthenticationType.access_token.value: {
+            "key": "accessTokenSettings",
+            "displays": [
+                ("Secret Reference", "secretRef"),
+            ],
+        },
+        DataFlowEndpointAuthenticationType.system_assigned.value: {
+            "key": "systemAssignedManagedIdentitySettings",
+            "displays": [
+                ("Audience", "audience"),
+            ],
+        },
+        DataFlowEndpointAuthenticationType.user_assigned.value: {
+            "key": "userAssignedManagedIdentitySettings",
+            "displays": [
+                ("Client ID", "clientId"),
+                ("Scope", "scope"),
+                ("Tenant ID", "tenantId"),
+            ],
+        },
+        DataFlowEndpointAuthenticationType.x509.value: {
+            "key": "x509CertificateSettings",
+            "displays": [
+                ("Secret Ref", "secretRef"),
+            ],
+        },
+        DataFlowEndpointAuthenticationType.service_account_token.value: {
+            "key": "serviceAccountTokenSettings",
+            "displays": [
+                ("Audience", "audience"),
+            ],
+        },
+        DataFlowEndpointAuthenticationType.sasl.value: {
+            "key": "saslSettings",
+            "displays": [
+                ("Type", "saslType"),
+                ("Secret Ref", "secretRef"),
+            ],
+        },
+        DataFlowEndpointAuthenticationType.anonymous.value: {
+            "key": "anonymousSettings",
+            "displays": [],
```

Comment:
is there no property under anonymousSettings?

Reply:
Currently there are no defined properties - it's just an empty object with no description in the CRD
![image](https://github.com/user-attachments/assets/10d4739b-50a2-4830-ab25-6be2a963122d)